ios/RNTPTDocumentView.m

```diff
@@ -1317,6 +1317,36 @@ - (void)importAnnotations:(NSString *)xfdfString
     if (!updateSuccess || error) {
         @throw [NSException exceptionWithName:NSGenericException reason:error.localizedFailureReason userInfo:error.userInfo];
     }
+    
+    return [self getAnnotationFromXFDF:xfdfString];
+
+}
+
+-(NSMutableArray<NSDictionary *> *)getAnnotationFromXFDF:(NSString *)xfdfString
+{
+    NSMutableArray<NSDictionary *> *annotations = [[NSMutableArray alloc] init];
+    PTFDFDoc *fdfDoc = [PTFDFDoc CreateFromXFDF:xfdfString];
```

Comment:
Should wrap the rest of this method in a try-catch to keep any exceptions local:
```suggestion
    @try {
        PTFDFDoc *fdfDoc = [PTFDFDoc CreateFromXFDF:xfdfString];
```